src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
+
+const widenAncestors = (el) => {
+  while ((el = el.parentElement) && !el.classList.contains('content_block')) {
```

Comment:
Not strictly, but I bailed at the content block since images are ultimately inside content blocks - no need to keep walking up the DOM past that point.